### ISSUE TITLE
Initial scan with virtual filesystem enabled starts from scratch if it's interrupted

### ISSUE THREAD

**Issue by wonx** [nextcloud/desktop#3914]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments. 

### Expected behaviour
When setting up Nextcloud with virtual filesystem enabled, it tries to scan every single folder in Nextcloud. This can take a while (hours or days) since TB of data comprising hundreds of thousands of files must be scanned for changes.

Sometimes it's not possible to leave a computer on for such a longs periods, or sometimes there's a small network downtime that interrupts that scan. In those case, I'd expect that the scan continues at the point where it left.

Moreover, I noticed that the "symbolic links" (probably not the correct term) to the files stored in the server are not created until all files have been checked for changes once, so it's not possible to access any of them until the very end. It would be more convenient if they were created on the go.

### Actual behaviour
The initial scan starts from scratch once again. In my case, I have to make sure that the computer that is going to use nextcloud is not turned off for ~15 hours, and make sure there will be network interruptions or nothing accesses the Nextcloud folder, since these things might trigger it to start once again from scratch.

Once workaround that I had used in the past was to exclude the largest folders first, by editing the ignored files options. But since that configuration is also saved in the nextcloud folder, and it's often still not available, it is not until the first scan has finished that you can edit that file.

### Steps to reproduce
1. Install Desktop client. 
2. Configure your account.
3. Start the initial sync.
4. Pause and resume the sync. 

### Client configuration
Client version: 3.3.5, 

Operating system: both in Windows 10 and Ubuntu 20.04.

OS language: English

Installation path of client: C:\Program Files\Nextcloud

### Server configuration

Nextcloud version: 22.1.0

Storage backend (external storage): sshfs

**Comment by wonx** [comment]
This is still a current issue.

**Comment by wonx** [comment]
I can confirm that this behavior also happens in MacOS Monterrey. I have been trying to do the initial sync in a macbook pro, and for reason, if it gets interrupted, it starts all over again. So far it's been syncing for more than 6 hours, but most of the time it's just checking the same folders over and over again.

**Comment by mgallien** [comment]
@wonx let me reopen
should not have been closed sorry

**Comment by wonx** [comment]
bump

**Comment by wonx** [comment]
bump

**Comment by wonx** [comment]
bump

**Comment by wonx** [comment]
I can confirm this issue is still present with the client v3.4.4. I reinstalled the operating system, and now the initial sync started. I estimate that it will take 10-15 uninterrupted hours to complete.

**Comment by tomdereub** [comment]
Bump.

**Comment by wonx** [comment]
This bug is still present.

**Comment by wonx** [comment]
Hi, can we reopen this bug? It's still present.

**Comment by wonx** [comment]
This bug is still present.

**Comment by wonx** [comment]
The bug is still present.

**Comment by wonx** [comment]
This is still a current issue with client 3.6.0. Please reopen.

**Comment by wonx** [comment]
Bump!

**Comment by KaeTuuN** [comment]
Hi, can you reopen this bug again? It's still an issue.

**Comment by KaeTuuN** [comment]
And open it again...

**Comment by wonx** [comment]
bump

**Comment by wonx** [comment]
Reopen, please, this issue is still current.

**Comment by wonx** [comment]
Hi, this is my bimonthly reopen petition 😇

**Comment by wonx** [comment]
😕

**Comment by wonx** [comment]
Hi. Can we keep this issue open until someone has looked at it?

**Comment by Rello** [comment]
Hello,

thank you for opening the request.
The client has improved a lot since the ticket was opened.
Can you support us and check if it is still relevant? In this case we appreciate a new ticket or a comment here.
Thank you for your support
